web/modules/custom/helloworld/src/Controller/HelloController.php

```diff
@@ -0,0 +1,33 @@
+<?php 
+
+namespace Drupal\helloworld\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Class for handling the routes of helloworld module.
+ */
+class HelloController extends ControllerBase {
+    /**
+   * Display the markup.
+   *
+   * @return array
+   *   Return markup array.
+   */
+  public function content() {
+    $user = \Drupal::currentUser()->getAccountName();
```

Comment:
We don't use services like this in class files.
Please go through the Drupal docs on Services and Dependency injection.